h2o-k8s/src/test/java/water/k8s/KubernetesEmbeddedConfigProviderTest.java

```diff
@@ -32,6 +32,7 @@ public void testActiveOnK8SEnvVariablesSet(){
     
     @Test
     public void testInactiveOnK8SEnvVariablesSet(){
+        environmentVariables.clear("H2O_KUBERNETES_SERVICE_DNS");
         kubernetesEmbeddedConfigProvider.init();
         assertFalse(kubernetesEmbeddedConfigProvider.isActive());
```

Comment:
We're testing for **inactivity** here. If the env var is not cleared, it will actually return true.